test_requirements.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+cloudpickle>=0.3.1
```

Comment:
I guess we need to bump this to 0.3.2 to have their fix of removing the \_\_transient\_\_ attribute